.github/workflows/publish-docker.yml

```diff
@@ -44,10 +44,14 @@ jobs:
         run: echo "${{ secrets.DOCKER_ACCESS_TOKEN }}" | docker login --username "${{ secrets.DOCKER_USERNAME }}" --password-stdin
       - name: Build and push image to Docker Hub (staging)
         run: |
+          BRANCH_NAME=$(echo "${{ github.ref }}" | sed -e "s/refs\/heads\///g")
```

Comment:
For the sake of consistency, please use all lowercase variables. We use all uppercase for environment variables only. Thus, `branch_name`, `build_tag`.